File: 0001 Code Chef/0015 Election Hopes/implementation/0011 lua/election_hopes.lua
string_input = io.read()
string_array = {}
count = 1

for i in string.gmatch(string_input,"%S+") do
    string_array[count] = i 
    count = count + 1
end

x = tonumber(string_array[1])
y = tonumber(string_array[2])

if(x >= 2*y) then
    print("YES")
else
    print("NO")
end

File: 0001 Code Chef/0015 Election Hopes/implementation/0006 php/election_hopes.php
<?php
    $string_input = fgets(STDIN);
    $string_array = explode(' ',$string_input);
    $x = (int) $string_array[0];
    $y = (int) $string_array[1];
    if($x >= $y*2){
        echo "YES\n";
    }else{
        echo "NO\n";
    }
?>